src/main/java/cz/xtf/radanalytics/util/configuration/RadanalyticsConfiguration.java

```diff
@@ -10,6 +10,7 @@
 	private static final String IMAGE_DOCKER_WEBDRIVER_HEADLESS_CHROME = "rad.web-driver.docker.image.chrome";
 	private static final String IMAGE_DOCKER_WEBDRIVER_HEADLESS_FIREFOX = "rad.web-driver.docker.image.firefox";
 	private static final String IMAGE_DOCKER_WEBDRIVER_VERSION = "rad.web-driver.docker.image.version";
+	private static final String BROWSER_NAME = "rad.web-driver.browser.name";  //chrome, firefox, phantomjs
```

Comment:
please rename this to WEBDRIVER_BROWSER_NAME, the same for the method getBrowserName()